spec/controllers/api/v1/courses_controller_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::CoursesController, type: :controller do
+  describe "index" do
+    it "render service unavailable" do
+      allow(controller).to receive(:index).and_raise(PG::ConnectionBad)
+      allow(controller).to receive(:authenticate)
+
+      get :index
+      expect(response).to have_http_status(:service_unavailable)
+      json = JSON.parse(response.body)
+      expect(json). to eq(
+        'code' => 503, 'status' => 'Service Unavailable'
```

Comment:
Can this be written as `code: 503, status: 'Service Unavailable'` ?

Fat arrows went out of fashion in Ruby 1.8

Reply:
argh, now i got to contend with catwalks and  fashion sense as well

:cry:

Reply:
Not a blocker. More of an FYI ;-)